app/assets/javascripts/past_teams.js

```diff
@@ -0,0 +1,16 @@
+$(function() {
+  var $selectedYear = $('select#past_teams');
```

Comment:
This adds a lot of non-default behavior to this select element. My personal guideline for this is usually: if I use Javascript to add behavior to an element, use data attributes, in this case something like`data-behavior="switch-teams-year"` and leave classes and ids fully for anything related to CSS and styling.

I even added some JS code to the Teams App once where I use it. Have a look at [city.js](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/app/assets/javascripts/city_fields.js) and the view files associated with it for inspiration.

Aside from that: I think it's sematically speaking slightly wrong to use a `select` tag here, since we're not in a form... What do you think?